### PR TITLE
Use singularity version 3.3.0 in CI and vendor

### PR DESCRIPTION
**Pull request by sashayakovtseva**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -29,7 +29,7 @@ require (
 	github.com/sirupsen/logrus v1.2.0 // indirect
 	github.com/stretchr/testify v1.2.2
 	github.com/sylabs/scs-library-client v0.3.0
-	github.com/sylabs/singularity v0.0.0-20190721223548-90e847a3fc6a
+	github.com/sylabs/singularity v0.0.0-20190725211753-a066ffcb65c2
```
Comment (cali4888) [inline go.mod]: can't we bind to the specific version?

Comment (sashayakovtseva) [reply go.mod]: My assumption is that `go mod` will change version to commit here because in Singularity repo there are non semver tags.

Comment (cali4888) [reply go.mod]: doesn't it look like semver tag? https://github.com/sylabs/singularity/releases/tag/v3.3.0

Comment (cali4888) [reply go.mod]: :(

**Pull request merged**